deis-dev/manifests/deis-builder-rc.yaml

```diff
@@ -37,7 +37,13 @@ spec:
             - name: minio-user
               mountPath: /var/run/secrets/object/store
               readOnly: true
+            - name: builder-key-auth
+              mountPath: /var/run/secrets/api/auth
```

Comment:
could this be a more descriptive filename like `/var/run/secrets/builder-api/auth`?